PusherMainView/PusherMainView/pushtypes.plist

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is there some specific reason to store push types in plist instead of enum for example?

Reply:
@imaginaris Yes it is easy to maintain.

Reply:
Hmm ok.